axelor-human-resource/src/main/java/com/axelor/apps/hr/web/timesheet/TimesheetController.java

```diff
@@ -245,30 +245,31 @@ public void cancel(ActionRequest request, ActionResponse response)  {
 	}
 	
 	//action called when confirming a timesheet. Changing status + Sending mail to Manager
-	public void confirm(ActionRequest request, ActionResponse response) throws AxelorException{
+	public void confirm(ActionRequest request, ActionResponse response) {
 
-		try{
+		try {
 			Timesheet timesheet = request.getContext().asType(Timesheet.class);
 			timesheet = timesheetRepositoryProvider.get().find(timesheet.getId());
 			TimesheetService timesheetService = timesheetServiceProvider.get();
-			
+
 			timesheetService.confirm(timesheet);
 
 			Message message = timesheetService.sendConfirmationEmail(timesheet);
-			if(message != null && message.getStatusSelect() == MessageRepository.STATUS_SENT)  {
+			if (message != null && message.getStatusSelect() == MessageRepository.STATUS_SENT) {
 				response.setFlash(String.format(I18n.get("Email sent to %s"), Beans.get(MessageServiceBaseImpl.class).getToRecipients(message)));
-			} 
-			
-		}  catch(Exception e)  {
-			TraceBackService.trace(response, e);
-		}
-		finally {
+			} else {
```

Comment:
Seems be strange change. If no email to send mode is configured, we will alert wrongly.